recipe/meta.yaml

```
@@ -1,52 +1,56 @@
 {% set name = "pytorch-lightning" %}
-{% set version = "1.9.0" %}
+{% set version = "1.9.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5b75fe936d16ef86dae22ea1cb0a73db281605cade682c0ef44e6508a99a0b37
+  sha256: 479164caea190d49ee2a218eef7e001888be56db912b417639b047e8f9ca8a07
 
 build:
-  number: 1
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # TODO: this upper bound is because of PyTorch issue with dataclass
+  skip: True  # [py<37 or py>311]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
 
 requirements:
   host:
     - pip
-    - python >=3.7, <3.11 # TODO: this upper bound is because of PyTorch issue with dataclass
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.7, <3.11 # TODO: this upper bound is because of PyTorch issue with dataclass
+    - python
     - packaging >=17.1
     - numpy >=1.17.2
     - pyyaml >=5.4
     - pytorch >=1.10.0
     - tqdm >=4.57.0
-    - fsspec[http] >2021.06.0
+    - fsspec >2021.06.0
     - torchmetrics >=0.7.0
     - typing-extensions >=4.0.0
-    - lightning-utilities >=0.4.2
+    - lightning-utilities >=0.6.0.post0
 
 test:
   imports:
     - pytorch_lightning
-  commands:
-    - pip check
   requires:
-    - python >=3.7, <3.11 # TODO: this upper bound is because of PyTorch issue with dataclass
+    - python >=3.7,<3.11 # TODO: this upper bound is because of PyTorch issue with dataclass
```

Shouldn't this be just python (since this is not noarch anymore)
```suggestion
    - python
```